Compilation fails on armhf

Building the package fails in a vendored library:

https://launchpad.net/ubuntu/+source/google-osconfig-agent/20210112.00-0ubuntu1/+build/20816292

https://launchpadlibrarian.net/517195733/buildlog_ubuntu-hirsute-armhf.google-osconfig-agent_20210112.00-0ubuntu1_BUILDING.txt.gz

```
# github.com/GoogleCloudPlatform/osconfig/vendor/cos.googlesource.com/cos/tools.git/src/pkg/cos
src/github.com/GoogleCloudPlatform/osconfig/vendor/cos.googlesource.com/cos/tools.git/src/pkg/cos/env_reader.go:65:82: cannot use c.uname.Release[:] (type []uint8) as type []int8 in argument to charsToString
```

--- a/packages/cos.go
+++ b/packages/cos.go
@@ -12,9 +12,10 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-// Only build for linux but not s390x ppc64le ppc64.
+// Only build for linux but not on unsupported architectures.
 
-// +build linux,!s390x,!ppc64le,!ppc64
+// +build linux
+// +build 386 amd64
 
 package packages
 

--- a/packages/cos_stub.go
+++ b/packages/cos_stub.go
@@ -12,10 +12,9 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-// Stub for s390x ppc64le ppc64 linux builds.
+// Stub for linux builds.
 
-// +build linux
-// +build s390x ppc64le ppc64
+// +build linux,!386,!amd64
 
 package packages
 

--- a/packages/cos_test.go
+++ b/packages/cos_test.go
@@ -12,7 +12,8 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-// +build linux,!s390x,!ppc64le,!ppc64
+// +build linux
+// +build 386 amd64
 
 package packages
 

